app.rb

```diff
@@ -1,8 +1,21 @@
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  get '/' do
+    'Welcome to Chitter!'
+  end
+
+  get '/peeps' do
+    #get the peeps data from the database
+    #display the peeps int he view
+    @peeps = Peeps.all
+    erb :peeps
+  end
+
+  post '/add_peep' do
+    Peeps.add_peep(message: params['message'])
+    redirect '/peeps'
```

Comment:
I think there's an opportunity here to separate the redirect into it's own GET request :)